workspaces/liferay-sample-workspace/poshi/build.gradle

```diff
@@ -6,4 +6,14 @@ dependencies {
 
 poshiRunner {
 	poshiPropertiesFile = file("poshi.properties")
+}
+
+repositories {
+	maven {
+		url "https://repository-cdn.liferay.com/nexus/content/groups/public"
+	}
+
+	maven {
+		url "https://repository.liferay.com/nexus/content/groups/public"
```

Comment:
We should be able to remove this one.

Reply:
I'll try it out.  Thanks for the heads up!